src/util.py

```diff
@@ -40,6 +40,29 @@ def get_quarter_collections(year, quarter, collection_type):
         abort(404, message="Not Found(404): Database collection entry {} doesn't exist".format(collection_name))
     return coll
 
+def get_contact_collections():
+    """Get the collection from db according to defined quarter and the type of collection requested.
+
+    Args:
+        year: the year of the requested quarter
+        quarter: the requested quarter
+        collection_type: str, either "courses" or "departments"
+    Returns:
+        The collection received from db
+
+    """
+    env_config = ConfigParser()
```

Comment:
Please move line 54-59 to a separate function, maybe call get_mongo_client() so get_contact_collections() and get_db() and both use get_mongo_client().